sql/moz-fx-data-shared-prod/telemetry_derived/cohort_daily_statistics_v1/metadata.yaml

```diff
@@ -14,8 +14,13 @@ labels:
 scheduling:
   dag_name: bqetl_analytics_aggregations
   date_partition_parameter: cohort_date
-  arguments: ['--append_table']
+  arguments: ["--append_table"]
+  parameters: ["submission_date:DATE:{{ds}}"]
```

Comment:
Append also makes sense for active_users_aggregates. I will add that in a separate PR.